nvmf-autoconnect/udev-rules/65-persistent-net-nbft.rules.in

```diff
@@ -0,0 +1,2 @@
+# Avoid renaming nbft$X interfaces
+SUBSYSTEM=="net", ACTION=="add", ENV{INTERFACE}=="nbft*", NAME:="%E{INTERFACE}"
```

Comment:
I thought the we should not use `ACTION==add` anymore. Instead something like `ACTION!=remove` so that `change` is also handled (can't find the reference anymore though).

Reply:
That's because for block devices we need to handle both `add` and `change`. I think it's different for net devices, it particular for rules that are used to change the device name, like this one. The existing rules I've seen for network device naming all use `add` only.

Reply:
This is from Lennart on this topic:
> rules files should always use a logic where instead of listing positives, they should list negatives. i.e. create your SYMLINKS= props in all cases but in "remove". This is what udev's/systemd's own rules files do, and is what should be done in 3rd party files too.

and after looking through systemd's udev rules I see the mentioned pattern:

```
# do not edit this file, it will be overwritten on update

SUBSYSTEM!="net", GOTO="net_setup_link_end"

IMPORT{builtin}="path_id"

ACTION=="remove", GOTO="net_setup_link_end"

IMPORT{builtin}="net_setup_link"

NAME=="", ENV{ID_NET_NAME}!="", NAME="$env{ID_NET_NAME}"

LABEL="net_setup_link_end"
```

Reply:
as we running low on time, let's go with your version and fix this up later.